man/man8/killsnoop.8

```diff
@@ -27,7 +27,10 @@ Print usage message.
 Only print failed kill() syscalls.
 .TP
 \-p PID
-Trace this process ID only (filtered in-kernel).
+Trace this process ID only which is the sender of signal (filtered in-kernel).
+.TP
+\-t PID
+Trace this target process ID only which is the receiver of signal (filtered in-kernel).
```

Comment:
-t is commonly used for tid. So I would like this option to be -r/--rpid to imply it is for signal receiver.

Reply:
what about '-T/-tpid'?

Reply:
'-T/-tpid' is okay to me.